### ISSUE TITLE
Error when processing custom SMPL files

### ISSUE THREAD

**Issue by nshreyasvi** [MarilynKeller/HIT#2]
Hello, The work is really impressive and I was interested in testing on custom SMPL meshes but when I run the inference script, I am getting the following error:
```
RuntimeError: The expanded size of the tensor (63) must match the existing size (32) at non-singleton dimension 1.  Target sizes: [1, 63].  Tensor sizes: [32]
```
Are the SMPL inputs to the inference model in a different format? I tested using SMPL, SMPLH and SMPLX and get the same error in them. 
I have attached SMPL file here:
[106.zip](https://github.com/user-attachments/files/17125377/106.zip)

**Comment by MarilynKeller** [comment]
From the error it looks like the pose parameters  you provide are in an array of shape 32. SMPL has one root joint plus 23 joints so the body pose parameters should be of shape 69. Where did you get this SMPL file?

**Comment by nshreyasvi** [comment]
Hello, Thank you for your response. I generated them using the [SMPLify-X](https://github.com/vchoutas/smplify-x/) library by inputting the RGB image and pose keypoints (extracted through OpenPose). I wanted to know if there is another method that might work better for image to SMPL generation as when I use the SMPLify-X library, I am getting the format error both while using HIT and also when I tried to implement SKEL ([Issue](https://github.com/MarilynKeller/SKEL/issues/17)).

**Comment by MarilynKeller** [comment]
You can try HMR2.0 to infer SMPL from images: 
https://github.com/russoale/hmr2.0

**Comment by nshreyasvi** [comment]
Hello, Thank you for your response. I was able to fix it. 

- I used [MMHuman3D](https://github.com/open-mmlab/mmhuman3d) to generate SMPL (using HMR method and not SMPLify) and generated an output `.pkl` file by modifying the demo scripts there.
- `body_pose` had shape `['frame_count', 23, 3)`.  I then reshaped it so it matches the `(1, 69)` that the model expects. 
- I further changed the `hit/utils/data.py` file as it expects a (1,63) shape file and changed it to (1,69) on line 27
- I then added the like `torch_param['transl'] = None` as the SMPL file generated does not have it.

I was able to generate the .obj files using this method. Thank you so much.
